stream-video-android-compose/src/main/kotlin/io/getstream/video/android/compose/ui/components/call/ringing/outgoingcall/OutgoingCallContent.kt

```diff
@@ -58,19 +58,19 @@ import io.getstream.video.android.mock.mockParticipantList
 public fun OutgoingCallContent(
     modifier: Modifier = Modifier,
     call: Call,
-    isVideoType: Boolean,
+    isVideoType: Boolean = true,
```

Comment:
I think giving this value automatically here is not a good idea - because Incoming/Outgoing calls have different UI/UX (You can check the out in our internal design guidelines) depending on this value.

Reply:
True. I removed the "common call UI" commit from this PR in the end.